README.md

```diff
@@ -163,7 +163,19 @@ Future:
 ## Notes
 * Probably think we should use [Click](https://github.com/pallets/click)
 * Need to keep in mind that the APIs are currently not fixed (we're making best efforts to not change them too much but they are liable to change) The CLI does need to be fixed so it probably needs to send a UserAgent API header to let us know which version of the API they're using so that in future we can know not to break stuff that's going to affect a lot of people.
-​
+
+## DAFNI Endpoint Information
+The Swagger documents for the DAFNI components are available at the following links:
+* [DSS](https://app.swaggerhub.com/apis/tg137/DSS-API/1.0.0)
+* [NIMS](https://app.swaggerhub.com/apis/tg137/NIMS-API/1.0.0)
+* [Search and Discovery](https://app.swaggerhub.com/apis/tg137/S-AND-D-API/1.0.0)
+* [NIVS](https://app.swaggerhub.com/apis/tg137/nivs-api/1.0.0)
```

Comment:
This link doesn't work.
![image](https://user-images.githubusercontent.com/7328717/109792774-c6662700-7c0b-11eb-8ea3-7010918d34a9.png)

Reply:
Whoops, that one was private for some reason. Fixed now I think

Reply:
Yep, looks good. I'm happy.